scala/akkahttp/build.sbt

```diff
@@ -2,9 +2,9 @@ import sbtassembly.AssemblyPlugin.defaultShellScript
 
 name := "AkkaHttp"
 
-version := "0.0.1"
+version := "0.0.2"
 
-scalaVersion := "2.12.2"
+scalaVersion := "2.13.1"
```

Comment:
is their a way to set a more relax version of `scala` ? I mean using a range like `[2.13,2.14)` in order to be sure to stick to **2.13.x** but use the most up-to-date version in this minor

Reply:
Not that I know of.